resources/lang/en/thoughts/alive/general.json

```
@@ -113,7 +117,39 @@
             "Wonders how r_c is doing",
             "Is giving some advice to r_c",
             "Wonders what r_c is up to",
-            "Is playing a prank on r_c"
+            "Is playing a prank on r_c",
+			"Saw r_c the other day, but didn't get a chance to meow hello",
+			"Bickered about something trivial with r_c",
+			"Nods politely as r_c passes by",
+			"Trips over the same twig that r_c just tripped over",
+			"Watches r_c wind up for a sneeze, but it never happens",
+			"Asks r_c if {PRONOUN/m_c/poss} fur looks clean",
+			"Points out a constellation to r_c",
+			"Keeps trying to figure out what a bad smell in camp is with r_c",
```

```suggestion
			"Is trying to figure out where a bad smell is coming from with r_c",
```
I think this sounds a bit more natural